swagger/swagger.yaml

```diff
@@ -9,8 +9,8 @@ info:
   license:
     name: MIT
     url: https://opensource.org/licenses/MIT
-host: kb.phenoscape.org
-basePath: /api
+# host: kb.phenoscape.org
+# basePath: /api
```

Comment:
Is this on purpose?

Reply:
No, this was probably already there on swagger hub where I pasted this from. But I'll remove it.